modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -592,11 +607,21 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
                                 pad_type);
         }
 
-        if (hasBias() || fusedBias)
+        if (hasBias() || fusedBias || nodes.size() == 3)
```

Comment:
Bias can be also as input? 😕

Reply:
Sure))

Reply:
Do we have default backend support for it?

Reply:
This test contains bias `testONNXModels("conv_depthwise_variable_w", npy, 0, 0, false, false, 3);` and passed on OpenCV and nGraph backends.